WinImageBuilder.psm1

```diff
@@ -1177,6 +1234,7 @@ function New-WindowsCloudImage {
         }
         Generate-UnattendXml @xmlParams
         Copy-UnattendResources $resourcesDir $image.ImageInstallationType $windowsImageConfig.install_maas_hooks

```

Comment:
add the vmware tools path to the copy-unattended resources method, we shouldn't make a special method for vmware